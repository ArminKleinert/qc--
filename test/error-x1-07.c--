
section "data" 
{
    x:
    bits32;

}

foo (bits32 y) {

    y = bits32 [x aligned 3]; 
}


