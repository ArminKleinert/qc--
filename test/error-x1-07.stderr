Character 68-88 Error: alignment not a power of 2
